doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -42,16 +42,52 @@ toplevel with the command ``Coqloop.loop();;``.
 Batch compilation (coqc)
 ------------------------
 
-The ``coqc`` command takes a name *file* as argument. Then it looks for a
-file named *file*.v, and tries to compile it into a
-*file*.vo file (See :ref:`compiled-files`).
-
-.. caution::
-
-   The name *file* should be a regular Coq identifier as defined in Section :ref:`lexical-conventions`.
-   It should contain only letters, digits or underscores (_). For example ``/bar/foo/toto.v`` is valid,
-   but ``/bar/foo/to-to.v`` is not.
-
+The ``coqc`` command compiles a Coq proof script file with a ".v" suffix
+to create a compiled file with a ".vo" suffix.  (See :ref:`compiled-files`.)
+The components of the filename must be valid Coq identifiers as described in
+:ref:`lexical-conventions`; they should contain only letters, digits or
+underscores (_) with a ".v" suffix on the final component.
+For example ``/bar/foo/toto.v`` is valid, but ``/bar/foo/to-to.v`` is not.
```

Comment:
There is no restriction on the validity of the directory which contains a `.v` file:

```
$ coqc test\ with\ space/test.v 
```
works.

Reply:
This is not new text.  How does such a name get mapped to a module name?  I see such names as something to avoid even if they're permitted.

Reply:
You extended the existing text. The previous test only talked about the filename (i.e. last component). I don't know exactly how things work, so I cannot answer your question precisely.